docs/waf-rules.md

```diff
@@ -0,0 +1,84 @@
+---
+
+template:      article
+reviewed:      2022-03-01
+naviTitle:     WAF rules
+title:         Web Application Firewall
+excerpt:       Protect your App against common attacks and harmful requests
+group:         platform
+stack:         all
+dontList:      true
+dontIndex:     true
```

Comment:
Why dontList/dontIndex?

Reply:
With these frontmatter meta tags you can push a new page to production without being linked / indexed. It's documented in the README:

```
# don't show in list on homepage
dontList: true

# do not include in search, don't show in search results
dontIndex: true
```

In this case (PR workflow) I think it's not necessary. Better don't otherwise we forget to remove.